Source/ThirdParty/skia/CMakeLists.txt

```
@@ -3,6 +3,7 @@ set_property(DIRECTORY . PROPERTY FOLDER "skia")
 # Skia dependencies not used directly in WebKit.
 find_package(Freetype 2.9.0 REQUIRED)
 find_package(Fontconfig 2.13.0 REQUIRED)
+find_package(WebP REQUIRED COMPONENTS demux mux)
```

Here it should be enough to list the `mux` component, because the `demux` one is not used in the rest of the `CMakeLists.txt` file.

```suggestion
find_package(WebP REQUIRED COMPONENTS mux)
```

---

Right, I copy pasted and forgot to remove the demux component.